crates/router/src/types.rs

```diff
@@ -419,6 +419,7 @@ pub struct PaymentsCaptureData {
     pub multiple_capture_data: Option<MultipleCaptureRequestData>,
     pub connector_meta: Option<serde_json::Value>,
     pub browser_info: Option<BrowserInformation>,
+    pub metadata: Option<pii::SecretSerdeValue>,
```

Comment:
Please add comment on from where and what info this metadata holds?

Reply:
Done